Tensile/Source/lib/include/Tensile/PropertyMatching.hpp

```diff
@@ -150,10 +140,207 @@ namespace Tensile
             {
             }
 
+            virtual std::string distanceType() const
+            {
+                return Distance::Type();
+            }
+
             ReturnValue findBestKeyMatch(Key const& key, Transform transform) const
             {
                 const bool debug = Debug::Instance().printPropertyEvaluation();
+                const bool naive = Debug::Instance().naivePropertySearch();
+
+                if(naive)
+                {
+                    if(debug)
```

Comment:
Is debug set at runtime?

Reply:
Yes, it's set by one of the TENSILE_DB bits.